configuration/src/agent/signer.rs

```diff
@@ -40,6 +40,10 @@ impl FromEnv for SignerConf {
             return Some(SignerConf::HexKey(HexString::from_str(&signer_key).ok()?));
         }
 
+        if let Some(prefix) = default_prefix {
+            return SignerConf::from_env(prefix, None);
+        }
+
```

Comment:
A little confused. We pass in Option default prefix but say that if Option is `Some`, call the same function with default prefix `None`? Doesn't this make `SignerConf::from_env(prefix, None)` the same as `SignerConf::from_env(prefix, Some(...))`?

Reply:
Nvm misread, conflated the two `prefix` vars